gqltesting/testing.go

```diff
@@ -43,22 +43,22 @@ func RunTest(t *testing.T, test *Test) {
 		test.Context = context.Background()
 	}
 	result := test.Schema.Exec(test.Context, test.Query, test.OperationName, test.Variables)
-	// Verify JSON to avoid red herring errors.
-	got, err := formatJSON(result.Data)
-	if err != nil {
-		t.Fatalf("got: invalid JSON: %s", err)
-	}
-	want, err := formatJSON([]byte(test.ExpectedResult))
-	if err != nil {
-		t.Fatalf("want: invalid JSON: %s", err)
-	}
-
 	checkErrors(t, test.ExpectedErrors, result.Errors)
-
-	if !bytes.Equal(got, want) {
-		t.Logf("got:  %s", got)
-		t.Logf("want: %s", want)
-		t.Fail()
+	if test.ExpectedResult != "" {
```

Comment:
Can we ever expect the result to be empty string? If yes, can you, please, reverse this condition:
```go
if test.ExpectedResult == "" {
    // handle this case
}
// unindent the rest of the code
```